CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ project(TheoIDE VERSION 0.1.0 LANGUAGES CXX)
 set(QT_VERSION 6.7)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION})
+set(CPACK_PACKAGE_NAME TheoIDEApp-Setup)
```

Comment:
Why not name it like the project name (default). With this config, the application is named "TheoIDEApp-Setup" after installation, right?

Reply:
The application itself is still called TheoIDEApp but the Debian bundle is called TheoIDEApp-Setup, which also means you remove it with "dpkg -r TheoIDEApp-Setup". So yes it is probably better to just call it TheoIDEApp as well.